debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-i18n (3.15-1) unstable; urgency=low
```

Comment:
`3.15-0`?

Reply:
Apparently that usually starts at 1 (hence all the previous ones did)
```It is conventional to restart the debian_revision at 1 each time the upstream_version is increased.

The package management system will break the version number apart at the last hyphen in the string (if there is one) to determine the upstream_version and debian_revision. The absence of a debian_revision is equivalent to a debian_revision of 0.```